app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
what about `options` as the second argument, accepting optionally:

* logger
* description: such as "Unhandled VMware error", that we have on couple of places
* level: warn by default